### PR TITLE
fix: blocks UI elements on right side

### LINKED ISSUE #30879: Fullscreen mode: Unnecessary vertical scrollbar blocks UI elements on right side of screen
## Screenshot

chrome_j5mjQOUXzT
chrome_8UBAYWGVV9

## Description

When a graph is in fullscreen mode, there is an unnecessary vertical scrollbar that blocks some of the UI elements on the right side of the screen (i.e. the “Inv” button).  This vertical scrollbar remains visible regardless of the browser window size.

Expected: 

The scrollbar should not be present. Even if the scrollbar is present, the buttons on the corner should not be affected the way they are getting hidden.

Actual:

The scrollbar is present and obscures some page elements under it

### PR DESCRIPTION
**Pull request by samarsrivastav**
fix(dashboard): load charts correctly

Fixes #30879 
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
shown in the issue #30879 

### ADDITIONAL INFORMATION

- [x] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx [hunk 1]
```diff
@@ -76,6 +76,7 @@ const fullSizeStyle = css`
     z-index: 3000;
     left: 0;
     top: 0;
+    padding:10px;
```
[the remark this answers is not in the document]
Comment (samarsrivastav) [reply superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx:79]: 8px will also be working fine here!!

**Comment by samarsrivastav** [comment]
hey @rusackas !! can i connect with you on slack or something, i have some doubts regarding the setup of the application

**Comment by rusackas** [comment]
By all means, [slack](http://bit.ly/join-superset-slack) away... or start a GitHub Discussion and tag me there if you want more eyes on it.

**Review by rusackas — APPROVED** [review]

File: superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx [hunk 2]
```diff
@@ -76,6 +76,7 @@ const fullSizeStyle = css`
     z-index: 3000;
     left: 0;
     top: 0;
+    padding: ${theme.gridUnit * 2}px;
```
Comment (rusackas) [inline superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx:79]: Oops... just need to add a `const theme = useTheme();`  before the `const fullSizeStyle...` bit, and pull in `{ useTheme } from '@superset-ui/core'`

File: superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx [hunk 3]
```diff
@@ -32,6 +32,7 @@ import ResizableContainer from 'src/dashboard/components/resizable/ResizableCont
 import getChartAndLabelComponentIdFromPath from 'src/dashboard/util/getChartAndLabelComponentIdFromPath';
 import useFilterFocusHighlightStyles from 'src/dashboard/util/useFilterFocusHighlightStyles';
 import { COLUMN_TYPE, ROW_TYPE } from 'src/dashboard/util/componentTypes';
+import { useTheme } from '@superset-ui/core';
```
Comment (rusackas) [inline superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx:35]: Ahh, you can add this import to line 25, so you're not importing stuff from superset-ui/core twice (it's a listing error). Sorry, I should have been clearer. Also, sorry you even have to DO this stuff... but it's all stuff that sets things up better for the next person :)

File: superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx [hunk 4]
```diff
@@ -69,13 +70,14 @@ interface ChartHolderProps {
   setFullSizeChartId: (chartId: number | null) => void;
   isInView: boolean;
 }
-
+const theme=useTheme()
```
Comment (rusackas) [inline superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx:73]: Looks like this (and the  style const below it) might need to go within the function component... or there are other workarounds for that. Holler if you want any help here, I _might_ be able to push commits to your fork/branch

**Comment by samarsrivastav** [comment]
hey @rusackas , can you check it now

**Comment by sadpandajoe** [comment]
@samarsrivastav looks like there are some failures with pre-commit hooks. Mind running them?

**Comment by rusackas** [comment]
Thank you @samarsrivastav! Looks good!

**Pull request merged**